packs/feats/sword-light-wave.json

```
@@ -28,7 +28,26 @@
             "remaster": true,
             "title": "Pathfinder Lost Omens: Tian Xia Character Guide"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "RollOption",
+                "label": "PF2E.SpecificRule.SpiritWarrior.SpiritDamage",
+                "option": "deal-spirit-damage",
+                "toggleable": true
+            },
+            {
+                "key": "DamageAlteration",
+                "mode": "override",
+                "predicate": [
+                    "deal-spirit-damage"
+                ],
```

```suggestion
            {
                "key": "RollOption",
                "label": "PF2E.SpecificRule.SpiritWarrior.SpiritDamage",
                "option": "sword-light-wave-spirit",
                "toggleable": true
            },
            {
                "key": "DamageAlteration",
                "mode": "override",
                "predicate": [
                    "sword-light-wave-spirit"
                ],
```
Best to be more specific in the roll options.